### ISSUE TITLE
add MacOS Firefox Theme

### ISSUE THREAD

**Issue by chunky-milk** [techcoder20/MacOSBigSurThemeConverter#8]
I found [this project](https://github.com/AdamXweb/WhiteSurFirefoxThemeMacOS), which is a theme for Firefox that looks like Safari, could you please add it?

(To install the latest firefox you can use the instructions [here](https://github.com/chunky-milk/pi-bashscripts-files/blob/main/Firefox.md))

**Comment by Itai-Nelken** [comment]
I used https://github.com/vinceliuice/WhiteSur-gtk-theme/tree/master/src%2Fother%2Ffirefox on tge firefox in the repos and it worked fine.

**Comment by techcoder20** [comment]
Good Idea This will be a nice addition to the theme. I will try to add it :)
